### PR TITLE
Update AboutUs.md

### LINKED ISSUE #10: Update Yi Lin's info for AboutUs.md

### PR DESCRIPTION
**Pull request by yleeyilin**
fixes #10

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -5,19 +5,19 @@ title: About Us
 
 We are a team based in the [School of Computing, National University of Singapore](http://www.comp.nus.edu.sg).
 
-You can reach us at the email `seer[at]comp.nus.edu.sg`
+You can reach us at the email `cs2103t_w10_2[at]comp.nus.edu.sg`
```
Comment (jannaleong) [inline docs/AboutUs.md:8]: This email doesnt seem to be working. is there a valid email we can change it to?

File: docs/team/yleeyilin.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+---
+layout: page
+title: Yi Lin's Project Portfolio Page
+---
+
+### Project: AddressBook Level 3
```
Comment (jannaleong) [inline docs/team/yleeyilin.md:6]: Should we make this project description specific to our AB3 aka poochplanner

File: docs/team/yleeyilin.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+Given below are my contributions to the project.
+
+* **New Feature**: Added the ability to undo/redo previous commands.
+  * What it does: allows the user to undo all previous commands one at a time. Preceding undo commands can be reversed by using the redo command.
+  * Justification: This feature improves the product significantly because a user can make mistakes in commands and the app should provide a convenient way to rectify them.
+  * Highlights: This enhancement affects existing commands and commands to be added in future. It required an in-depth analysis of design alternatives. The implementation too was challenging as it required changes to existing commands.
+  * Credits: *{mention here if you reused any code/ideas from elsewhere or if a third-party library is heavily used in the feature so that a reader can make a more accurate judgement of how much effort went into the feature}*
+
+* **New Feature**: Added a history command that allows the user to navigate to previous commands using up/down keys.
```
Comment (jannaleong) [inline docs/team/yleeyilin.md:18]: what is this history command?

**Review by jannaleong — COMMENTED** [review]
Overall p good but I have some questions.

File: docs/team/yleeyilin.md (hunk 2 above)
Comment (yleeyilin) [reply docs/team/yleeyilin.md:6]: Agreed, ill make the required changes.

File: docs/team/yleeyilin.md (hunk 3 above)
Comment (yleeyilin) [reply docs/team/yleeyilin.md:18]: This is a template from the readme that they gave us.

File: docs/AboutUs.md (hunk 1 above)
Comment (yleeyilin) [reply docs/AboutUs.md:8]: I think it is best we leave it out since there is no valid email we can change to.

**Review by jannaleong — APPROVED** [review]
LGTM

**Pull request merged**